portfolio/src/main/webapp/js/script.js

```diff
@@ -18,23 +18,92 @@
 */
 filterGallery('all');
 function filterGallery(galleryFilter) {
-    var documents = document.getElementsByClassName("gallery");
-    
-    if (galleryFilter == "all")
-        galleryFilter = "";
-    
-    for (var index = 0; index < documents.length; index++) {
-        documents[index].className = documents[index].className.replace("show-gallery-img", "");
+    let documents = document.getElementsByClassName('gallery');
+
+    if (galleryFilter == 'all') galleryFilter = '';
+
+    for (let index = 0; index < documents.length; index++) {
+        documents[index].className = documents[index].className.replace(
+            'show-gallery-img',
+            ''
+        );
         if (documents[index].className.includes(galleryFilter)) {
-            documents[index].className += " show-gallery-img"
+            documents[index].className += ' show-gallery-img';
         }
     }
 }
 
 function truthLieClicked(truth) {
-    if (truth == "truth") {
-        alert("Truth!");
+    if (truth == 'truth') {
+        alert('Truth!');
     } else {
-        alert("Lie! Try again ...");
+        alert('Lie! Try again ...');
     }
-}
\ No newline at end of file
+}
+
+/** Fetches the list of comments from the Servlet */
+function listComments() {
+    fetch('/list-comments')
+        .then((response) => response.json())
+        .then((comments) => {
+            const commentListElement = document.getElementById('comments-list');
+            comments.forEach((comment) => {
+                commentListElement.appendChild(createCommentElement(comment));
+            });
```

Comment:
Curious how this looks from the UX perspective - if your list is really long, this might look funky as comments load! Instead, you might want to create all the comment elements, and then append them all at once.